internal/sdk/cloudian/sdk.go

```diff
@@ -87,23 +87,22 @@ func fromInternal(g groupInternal) Group {
 	}
 }
 
-type User struct {
-	UserID  string `json:"userId"`
-	GroupID string `json:"groupId"`
-}
+type UserType string
 
-type userInternal struct {
-	UserID   string `json:"userId"`
-	GroupID  string `json:"groupId"`
-	UserType string `json:"userType"`
+const (
+	UserTypeSystemAdmin UserType = "SystemAdmin"
+	UserTypeGroupAdmin  UserType = "GroupAdmin"
+	UserTypeUser        UserType = "User"
+)
+
+type UserID struct {
+	GroupID string `json:"groupId"`
+	UserID  string `json:"userId"`
 }
 
-func toInternalUser(u User) userInternal {
-	return userInternal{
-		UserID:   u.UserID,
-		GroupID:  u.GroupID,
-		UserType: "User",
-	}
+type User struct {
+	UserID   `json:",inline"`
+	UserType UserType `json:"userType"`
```

Comment:
```suggestion
	UserType UserType `json:"type"`
```

Or will it break something?

Reply:
Must follow cloudian api